freqtrade/exchange/exchange.py

```diff
@@ -2055,6 +2055,42 @@ def _set_leverage(
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
 
+    def get_interest_rate(self) -> float:
+        """
+        Calculate interest rate - necessary for Margin trading.
+        """
+        return 0.0
+
+    def get_liquidation_price(
+            self,
+            pair: str,
+            open_rate: float,
+            amount: float,  # quote currency, includes leverage
+            leverage: float,
+            is_short: bool
+    ) -> Optional[float]:
+
+        if self.trading_mode in (TradingMode.SPOT, TradingMode.MARGIN):
```

Comment:
I think the `TradingMode.MARGIN` should be removed. It's not accurate, we just haven't written the code yet, so the OperationalException is the best outcome from this method